.github/workflows/pr-check.yml

```diff
@@ -111,10 +107,17 @@ jobs:
 
       - name: Build E2E test docker image
```

Comment:
You are right I tried it. But unfortunately, we have really restricted disk space. If I built it before minikube ran we have +2 GB on disk size. It is the main reason why I build the image after the API test. Because after Running the API test I have the possibility to remove e2e folder and free space (almost 2G).